sqlitedict.py

```diff
@@ -213,7 +214,7 @@ def __bool__(self):
     def iterkeys(self):
         GET_KEYS = 'SELECT key FROM "%s" ORDER BY rowid' % self.tablename
         for key in self.conn.select(GET_KEYS):
-            yield key[0]
+            yield _unconvertkey(key[0])
```

Comment:
Please use a more helpful function name, like _key_to_str (and conversely, _str_to_key).